CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 cmake_minimum_required (VERSION 3.5) # The minimum version of CMake necessary to build this project
 
+set( TRISYCL_VERSION_MAJOR 0)
```

Comment:
It makes me think that we do not have a versioning plan for triSYCL...
But your 0.1.0 looks like a good start.